### ISSUE TITLE
[Beta] Applied Accessibility - Tests fails to check for the closing </fieldset> tag

### ISSUE THREAD

**Issue by Asjas** [freeCodeCamp/freeCodeCamp#16422]
#### Challenge Name

https://beta.freecodecamp.org/en/challenges/applied-accessibility/wrap-radio-buttons-in-a-fieldset-element-for-better-accessibility

#### Issue Description

The issue with this challenge is that the test isn't checking for the corresponding closing tag for the `<fieldset>` tag. You can use a `</div>` tag as a closing tag and the test still passes. 

It should check for the `</fieldset>` closing tag.

The test also fails to check for a closing `</div>` tag and passes as well.

#### Browser Information

* Browser Name, Version: Chome, Version 63.0.3239.108 (Official Build) (64-bit)
* Operating System: Windows 10 Pro
* Mobile, Desktop, or Tablet: Desktop

#### Your Code

```js
<body>
  <header>
    <h1>Deep Thoughts with Master Camper Cat</h1>
  </header>
  <section>
    <form>
      <p>Sign up to receive Camper Cats blog posts by email here!</p>
      <label for="email">Email:</label>
      <input type="text" id="email" name="email">
      
      
      <!-- Add your code below this line -->
      <fieldset>
        <legend>What level ninja are you?</legend>
        <input id="newbie" type="radio" name="levels" value="newbie">
        <label for="newbie">Newbie Kitten</label><br>
        <input id="intermediate" type="radio" name="levels" value="intermediate">
        <label for="intermediate">Developing Student</label><br>
        <input id="master" type="radio" name="levels" value="master">
        <label for="master">Master</label>
      </div>
      <!-- Add your code above this line -->
      
      
      <input type="submit" name="submit" value="Submit">
    </form>
  </section>
  <article>
    <h2>The Garfield Files: Lasagna as Training Fuel?</h2>
    <p>The internet is littered with varying opinions on nutritional paradigms, from catnip paleo to hairball cleanses. But lets turn our attention to an often overlooked fitness fuel, and examine the protein-carb-NOM trifecta that is lasagna...</p>
  </article>
  <img src="samuraiSwords.jpeg" alt="">
  <article>
    <h2>Defeating your Foe: the Red Dot is Ours!</h2>
    <p>Felines the world over have been waging war on the most persistent of foes. This red nemesis combines both cunning stealth and lightening speed. But chin up, fellow fighters, our time for victory may soon be near...</p>
  </article>
  <img src="samuraiSwords.jpeg" alt="">
  <article>
    <h2>Is Chuck Norris a Cat Person?</h2>
    <p>Chuck Norris is widely regarded as the premier martial artist on the planet, and its a complete coincidence anyone who disagrees with this fact mysteriously disappears soon after. But the real question is, is he a cat person?...</p>
  </article>
  <footer>&copy; 2016 Camper Cat</footer>
</body>
```
#### Screenshot

![fieldset](https://user-images.githubusercontent.com/3828967/34639549-819c5ffc-f2ea-11e7-836a-61a7f5792db6.jpg)

**Comment by Pavel198007** [comment]
а от меня что нужно?15:14, 6 января 2018 г., "A-J Roos" <notifications@github.com>:




Challenge Name

https://beta.freecodecamp.org/en/challenges/applied-accessibility/wrap-radio-buttons-in-a-fieldset-element-for-better-accessibility
Issue Description

The issue with this challenge is that the test isn't checking for the corresponding closing tag for the <fieldset> tag. You can use a </div> tag as a closing tag and the test still passes.
It should check for the </fieldset> closing tag.
The test also fails to check for a closing </div> tag and passes as well.
Browser Information


Browser Name, Version: Version 63.0.3239.108 (Official Build) (64-bit)
Operating System: Windows 10 Pro
Mobile, Desktop, or Tablet: Desktop

Your Code

<body>
  <header>
    <h1>Deep Thoughts with Master Camper Cat</h1>
  </header>
  <section>
    <form>
      <p>Sign up to receive Camper Cats blog posts by email here!</p>
      <label for="email">Email:</label>
      <input type="text" id="email" name="email">
      
      
      
      <fieldset>
        <legend>What level ninja are you?</legend>
        <input id="newbie" type="radio" name="levels" value="newbie">
        <label for="newbie">Newbie Kitten</label><br>
        <input id="intermediate" type="radio" name="levels" value="intermediate">
        <label for="intermediate">Developing Student</label><br>
        <input id="master" type="radio" name="levels" value="master">
        <label for="master">Master</label>
      </div>
      
      
      
      <input type="submit" name="submit" value="Submit">
    </form>
  </section>
  <article>
    <h2>The Garfield Files: Lasagna as Training Fuel?</h2>
    <p>The internet is littered with varying opinions on nutritional paradigms, from catnip paleo to hairball cleanses. But lets turn our attention to an often overlooked fitness fuel, and examine the protein-carb-NOM trifecta that is lasagna...</p>
  </article>
  <img src="samuraiSwords.jpeg" alt="">
  <article>
    <h2>Defeating your Foe: the Red Dot is Ours!</h2>
    <p>Felines the world over have been waging war on the most persistent of foes. This red nemesis combines both cunning stealth and lightening speed. But chin up, fellow fighters, our time for victory may soon be near...</p>
  </article>
  <img src="samuraiSwords.jpeg" alt="">
  <article>
    <h2>Is Chuck Norris a Cat Person?</h2>
    <p>Chuck Norris is widely regarded as the premier martial artist on the planet, and its a complete coincidence anyone who disagrees with this fact mysteriously disappears soon after. But the real question is, is he a cat person?...</p>
  </article>
  <footer>&copy; 2016 Camper Cat</footer>
</body>
Screenshot



—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or mute the thread.


  
  




-- Отправлено из мобильного приложения Яндекс.Почты

**Comment by Asjas** [comment]
@Pavel198007 

I do not know what you mean by asking what I want from you. Can you explain?

Also you need to type in English and not Russian. So that everyone understands.

**Comment by ooade** [comment]
An added issue. Still passes even when the fieldset is not wrapped around the radio buttons
